src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.TodoTask;
+
+/**
+ * Handles saving to file and loading the file to the task list.
+ */
+public class Storage {
+    private static final String FILE_PATH = "./data/test.txt";
+
+    /**
+     * Saves the current list state to the statically set file.
+     *
+     * @param taskList Current task list
+     * @return 1 if operation if successful
+     */
+    public static int saveToFile(List<Task> taskList) {
+        FileWriter out = null;
+        //Check if folder/file is present, otherwise create file
+        File file = new File(FILE_PATH);
+        try {
+            String[] split = FILE_PATH.split("/");
```

Comment:
Perhaps, you can rename `split` to a more descriptive noun? There are a few other places where the variable name can be more descriptive, following the [code quality](https://nus-cs2103-ay2122s2.github.io/website/se-book-adapted/chapters/codeQuality.html) guide 😃

Reply:
thanks for the catch! will rename to something more descriptive